src/index.js

```diff
@@ -367,6 +368,15 @@ class Dropzone extends React.Component {
       autoComplete: 'off'
     }
 
+    const labelAttributes = {
+      style: {
+        width: '100%',
+        height: '100%',
+        margin: 0,
+        cursor: 'pointer'
```

Comment:
Not sure about this one especially without a way to customize it. Please remove cursor styling for now to make it non-breaking change.